src/stabilization_guide.md

```diff
@@ -194,3 +164,23 @@ if something { /* XXX */ }
 [Rust by Example]: https://github.com/rust-lang/rust-by-example
 [`Unstable Book`]: https://doc.rust-lang.org/unstable-book/index.html
 [`src/doc/unstable-book`]: https://github.com/rust-lang/rust/tree/master/src/doc/unstable-book
+
+## Lang team nomination
+
+When you feel the PR is ready for consideration by the lang team, you can [nominate the PR](https://lang-team.rust-lang.org/how_to/nominate.html) to get it on the list for discussion in the next meeting. You should also cc the other interacting teams to review the report:
+
+* `@rust-lang/types`, to look for type system interactions
+* `@rust-lang/compiler`, to vouch for implementation quality
+* `@rust-lang/opsem`, but only if this feature interacts with unsafe code and can create undefined behavior
+* `@rust-lang/libs-api`, but only if there are additions to the standard library
```

Comment:
Note that people who are not members of the org _cannot_ use team pings.

Reply:
I added a comment to recommend requesting reviewer to ping on their behalf.